### PR TITLE
Raise HTTP error if rate limited

### PR DESCRIPTION
**Pull request by dragon-dxw**
Generate a 420 "rate limited" error if that's the case.

### CODE REVIEW

File: api.rb [hunk 1]
```diff
@@ -25,20 +25,24 @@ def ensure_key_is_set
...
   json RedactedBreatheClient.absences(employee_id: params[:employee_id], after: params[:after])
+rescue RateLimited
+  halt(420, "Rate limited")
 end
 
 get "/sicknesses" do
   json RedactedBreatheClient.sicknesses(employee_id: params[:employee_id], after: params[:after])
-end
-
-get "/trainings" do
```
Comment (jdudley1123) [inline api.rb:38]: Let's put this is in a separate commit as it's unrelated to rate limiting

**Review by jdudley1123 — CHANGES_REQUESTED** [review]
Code changes look good! Can you make the commit and PR messages a bit more detailed?

File: api.rb (hunk 1 above)
Comment (dragon-dxw) [reply api.rb:38]: :+1:

**Review by jdudley1123 — APPROVED** [review]
